.github/workflows/coverage.yml

```diff
@@ -49,6 +49,7 @@ jobs:
         source ./.venv/bin/activate
         python -m pip install -U coverage
         python -m test.pythoninfo
+        export PYTHONPATH=./.venv/lib/python3.9/site-packages/coverage/fullcoverage
```

Comment:
This looks like it will break periodically, maybe we could find a way to avoid hardcoding the version?

Reply:
Agreed, although I couldn't find a simple alternative. I tried `.../python*/...` which does set the pythonpath to the right thing when echo'd, it doesn't seem to pick it up on the following command. If anyone can suggest a better alternative that would be great